app/controllers/modules/course/announcements_module.rb

```diff
@@ -5,8 +5,21 @@ class Course::AnnouncementsModule
     [
       {
         title: I18n.t('course.announcements.sidebar_title'),
-        unread: 0
+        unread: Course::AnnouncementsModule.get_unread_number(@current_user,@course)
       }
     ]
   end
+
+  private
+
+  def self.get_unread_number (current_user,course)
```

Comment:
Space missing after comma.<br>Do not put a space between a method name and the opening parenthesis.